Python/ast_opt.c

```diff
@@ -471,6 +471,39 @@ astfold_mod(mod_ty node_, PyArena *ctx_, int optimize_)
     return 1;
 }
 
+static int
+astfold_dedent(expr_ty node_, PyArena *arena, int optimize_)
+{
+    _Py_IDENTIFIER(dedent);
+    PyObject *dedent = PyUnicode_FromString("dedent");
+    if (!dedent) {
+        return 0;
+    }
+    if (asdl_seq_LEN(node_->v.Call.args) != 0 ||
+        asdl_seq_LEN(node_->v.Call.keywords) != 0 ||
+        node_->v.Call.func->kind != Attribute_kind) {
+        return 1;
+    }
```

Comment:
Likely dumb question, but isn't this block likely to be the one the triggers an early return most of the time?
Should it be the first in the function, allowing to not always have to create the `dedent` object ?